apollo-router/src/plugins/telemetry/tracing/jaeger.rs

```diff
@@ -101,29 +89,29 @@ impl TracingConfigurator for Config {
                     .build_async_agent_exporter(opentelemetry::runtime::Tokio)?;
                 Ok(builder.with_span_processor(
                     BatchSpanProcessor::builder(exporter, opentelemetry::runtime::Tokio)
-                        .with_batch_config(self.batch_processor.clone().into())
+                        .with_batch_config(batch_processor.clone().into())
                         .build()
                         .filtered(),
                 ))
             }
-            Endpoint::Collector {
-                endpoint,
-                username,
-                password,
-                ..
+            Config::Collector {
+                collector,
+                batch_processor,
             } => {
+                tracing::info!("configuring Jaeger tracing: {}", batch_processor);
```

Comment:
debug log to remove?

Reply:
All of the exporters log out this info, It's good to have because configuring the batch processor is pretty much if the router is under any kind of stress.

Reply:
If we feel this is an issue let's separate into a different PR. Current code preserves existing functionality.